src/explore-education-statistics-common/src/components/PageSearchForm.tsx

```diff
@@ -12,6 +12,7 @@ import classNames from 'classnames';
 import debounce from 'lodash/debounce';
 import React, { Component, ReactNode } from 'react';
 import Highlighter from 'react-highlight-words';
+import { logEvent } from '@frontend/services/googleAnalyticsService';
```

Comment:
This is in common but you're importing a service in from frontend (hence the build breaking)

Reply:
So we can import from @common -> @frontend but not vise-versa?
Should I move GA service to common?

Reply:
I'll wrap the common search component in the frontend and add the logging that way, instead of changing the common component like I have here